dev/AnimatedVisualPlayer/InteractionTests/AnimatedVisualPlayerTests.cs

```diff
@@ -42,8 +42,8 @@ class AnimatedVisualPlayerTests
         [ClassInitialize]
         [TestProperty("RunAs", "User")]
         [TestProperty("Classification", "Integration")]
-        [TestProperty("Platform", "Any")]
-        [TestProperty("DEPControlsTestSuite", "SuiteB")]
+        [TestProperty("TestPass:IncludeOnlyOn", "Desktop")]
+        [TestProperty("MUXControlsTestSuite", "SuiteB")]
```

Comment:
We can probably delete this entirely. It was used to split the tests into two "suites" to enable parallelization in Atlas. It is not used by Helix or by CatGates.